account_payment_order_vendor_email/views/account_payment_order_view.xml

```diff
@@ -0,0 +1,21 @@
+<odoo>
+    <record id="account_payment_order_manual_email_form" model="ir.ui.view">
+        <field name="name">account_payment_order_manual_email_form</field>
+        <field name="model">account.payment.order</field>
+        <field
+            name="inherit_id"
+            ref="account_payment_order.account_payment_order_form"
+        />
+        <field name="arch" type="xml">
+            <button name="action_done_cancel" position="after">
```

Comment:
There is no action_done_cancel button in parent view, it might be changed to action_uploaded_cancel

```
odoo.tools.convert.ParseError: while parsing /opt/odoo/kentest14-odoo/src/odoo-platform/odoo/external-src/bank-payment/account_payment_order_vendor_email/views/account_payment_order_view.xml:2, near
<record id="account_payment_order_manual_email_form" model="ir.ui.view">
        <field name="name">account_payment_order_manual_email_form</field>
        <field name="model">account.payment.order</field>
        <field name="inherit_id" ref="account_payment_order.account_payment_order_form"/>
        <field name="arch" type="xml">
            <button name="action_done_cancel" position="after">
                <button name="send_vendor_email" string="Send Email" type="object" class="oe_highlight" states="uploaded"/>
            </button>
        </field>
    </record>
```

Reply:
https://app.travis-ci.com/github/OCA/bank-payment/jobs/547322835#L1826
Travis fails